core/order.go

```diff
@@ -976,7 +976,7 @@ func (n *OpenBazaarNode) CalculateOrderTotal(contract *pb.RicardianContract) (*b
 			if !ok {
 				return big.NewInt(0), errors.New("invalid price value")
 			}
-			satoshis, err = n.getPriceInSatoshi(contract.BuyerOrder.Payment.AmountCurrency.Code, l.Item.PriceCurrency.Code, p)
+			satoshis, err = n.getPriceInSatoshi(contract.BuyerOrder.Payment.AmountCurrency.Code, l.Metadata.CryptoCurrencyCode, p)
```

Comment:
I don't believe this one should be cryptocurrency code... this block is just used for a regular listing.